src/controllers/orderController.js

```diff
@@ -23,6 +23,10 @@ const createOrder = async (req, res) => {
     const user = req.user; //contains user object
     console.log(user);
 
+    if (!user) {
+      throw new CustomError.UnauthenticatedError('User is not logged in');
+    }
```

Comment:
I believe the `authenticateUser` middleware on the route should already be forcing it to be the case that only logged in users will have their requests handled by this controller function?

Reply:
All the tests pass without this line so I think we can remove this :+1: